.github/dependabot.yml

```diff
@@ -13,6 +13,23 @@ updates:
     versioning-strategy: auto
     # Allow up to 10 open pull requests for updates to dependency versions
     open-pull-requests-limit: 10
+    # Group production and development (required and optional in the context of
+    # pyproject.toml) dependency updates when they are patch and minor updates,
+    # so we end up with less PRs being generated.
+    # Major updates are still managed, but they'll create one PR per
+    # dependency, as major updates are expected to be braking, it is better to
```

Comment:
`braking` -> `breaking`

Reply:
Same in the commit msg

Reply:
.. and every other copy of that text :laughing:

Reply:
It brakes the update... :P